packages/react-universal-provider/src/create-universal-provider.tsx

```diff
@@ -8,13 +8,13 @@ export interface UniversalProviderProps<Value> {
 
 export function createUniversalProvider<Value>(
   id: string,
-  Context: React.Context<Value | null>,
+  Context: React.Context<Value | undefined>,
```

Comment:
I am thinking maybe this should be `Context: React.Context<Value>`, let the consumer decided what this should be instead.